jobs/haproxy/spec

```diff
@@ -509,7 +509,7 @@ properties:
     description: "Number of dns queries to send to resolve a server name before giving up"
     default: 3
   ha_proxy.accept_proxy:
-    description: "Turned off by default. Enforces the use of the PROXY protocol for all incoming connections to all frontends. When enabled standard tcp connections to these port no longer work."
+    description: "Turned off by default. Enforces the use of the Proxy Protocol for all incoming connections to all frontends, with the exception of local requests to the health check endpoint, since monit does not support Proxy Protocol. When enabled standard TCP connections to these ports no longer work."
```

Comment:
either leave `PROXY` all uppercase, or change everywhere.

The [original docs](https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt) refer to it as `The PROXY Protocol`.

```suggestion
    description: "Turned off by default. Enforces the use of the PROXY Protocol for all incoming connections to all frontends, with the exception of local requests to the health check endpoint, since Monit does not support Proxy Protocol. When enabled standard TCP connections to these ports no longer work."
```